lib/LocalVerifiableCredentialStore.js

```diff
@@ -58,18 +59,25 @@ export class LocalVerifiableCredentialStore {
    * @param {boolean} options.displayable - Flag used for if credential is
    *   displable or not.
    * @param {string} options.parentId - ID of the parent credential.
+   * @param {string|Array} options.type - Type(s) of credential.
```

Comment:
Use square brackets to denote optional parameters.  Perhaps this applies to some of the other existing parameters?
```suggestion
   * @param {string|Array} [options.type] - Type(s) of credential.
```